ibis/pandas/core.py

```diff
@@ -226,9 +234,18 @@ def execute_until_in_scope(
                 op, op.value, expr.type(), aggcontext=aggcontext, **kwargs
             )
         }
-
+    # pre_executed_states is a list of states with same the length of
+    # computable_args, these states are passed to each arg
+    computed_localcontexts = compute_local_context(
```

Comment:
I don't like the name `computed_localcontexts`, I feel the word `computed` doesn't have any information here. Maybe call this `arg_localcontexts`?

Reply:
Also I wonder if this is clearer:
```
computed_localcontexts = compute_local_context(op, computed_args, localconntext)
```
Also not sure why are you passing clients here?

Reply:
Changed the name and fixed the function signature.